pkg/apis/config/feature_flags.go

```diff
@@ -92,6 +92,10 @@ const (
 	KeepPodOnCancel = "keep-pod-on-cancel"
 	// DefaultEnableKeepPodOnCancel is the default value for "keep-pod-on-cancel"
 	DefaultEnableKeepPodOnCancel = false
+	// EnableCelInWhenExpression is the flag to enabled CEL in WhenExpression
+	EnableCelInWhenExpression = "enable-cel-in-whenexpression"
+	// DefaultEnableCelInWhenExpression is the default value for EnableCelInWhenExpression
+	DefaultEnableCelInWhenExpression = false
```

Comment:
readability: for this and other use of CEL in code, please use uppercase for initials

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

```suggestion
	// EnableCELInWhenExpression is the flag to enabled CEL in WhenExpression
	EnableCELInWhenExpression = "enable-cel-in-whenexpression"
	// DefaultEnableCELInWhenExpression is the default value for EnableCELInWhenExpression
	DefaultEnableCELInWhenExpression = false
```

Reply:
Thanks! fixed